content/blog/2025/12/2025-12-19-twim.md

```diff
@@ -0,0 +1,268 @@
+> 
+> **Goals**
+> 
+> * Share Matrix knowledge, experiences, and best practices in a relaxed, human-scale setting
+> * Create space for learning, mentoring, and cross-project exchange
+> * Combine meaningful collaboration with the asian based matrix community
+> 
+> **Links** 
+> 
+> - [order your ticket](https://tickets.matrix-community.events/workation/asia-2026/)
+> - [matrix room for general infos](https://matrix.to/#/#workation-nation-matrix-thailand:datanauten.de)
+> - talk to  [yan](https://matrix.to/#/@yan:datanauten.de) or [nadine](https://matrix.to/#/@nadine:datanauten.de)
+> - [matrix room for other workation nation events](https://matrix.to/#/#workation-nation-general:datanauten.de)
+> - [cryptpad](https://cryptpad.fr/pad/#/2/pad/edit/iAvGs8PMOpaDkV7y0XijCR5g/) for more detailed information
+> - [Fossasia](https://eventyay.com/e/88882f3e)
+> - [Matrix @ Fosdem](https://matrix.org/blog/2025/12/fosdem-presence/)
+>
+> ![A picture of people working on laptops with a view on the beach. The title reads "Matrix Community Workation. Thailand 2026. From 2nd Februrary to 11th March, including FOSS Asia. Contact: yan@datanauten.de"](/blog/img/workation-26.png)
+
+## Matrix Federation Stats
+
+[Aine [etke.cc]](https://matrix.to/#/@aine:etke.cc) says
+
+> collected by [MatrixRooms.info](https://matrixrooms.info/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats) - an [MRS](https://github.com/etkecc/mrs) instance by [etke.cc](https://etke.cc?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats)
+> 
+> As of today, `11861` Matrix federateable servers have been discovered by matrixrooms.info, `3230` (`27.2%`) of them are publishing their rooms directory over federation.
+> The published directories contain `16731` rooms.
+> 
+> The most popular server software among the online servers is:
+> - synapse: `10137` (`85.5%`)
+> - conduit: `480` (`4.0%`)
+> - dendrite: `365` (`3.1%`)
+> - continuwuity: `324` (`2.7%`)
+> 
+> Stats timeline is available on [📊 MatrixRooms.info/stats](https://matrixrooms.info/stats/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats)
+> 
+> [🧩 Integrations with apps and servers](https://matrixrooms.info/integrations/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats) | [👉 How to add your server](https://matrixrooms.info/indexing/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats) | [🙅 How to remove your server](https://matrixrooms.info/deindexing/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats)
+
+## Dept of Ping
+
+Here we reveal, rank, and applaud the homeservers with the lowest ping, as measured by [pingbot](https://github.com/maubot/echo), a [maubot](https://github.com/maubot/maubot) that you can host on your own server.
+
+### [#ping:maunium.net](https://matrix.to/#/#ping:maunium.net)
+Join [#ping:maunium.net](https://matrix.to/#/#ping:maunium.net) to experience the fun live, and to find out how to add YOUR server to the game.
+
```

Comment:
where ping

Reply:
Still haven't set it up, do you have the script handy to add ping?